wiki/FitNesseRoot/HsacExamples/SlimTests/BrowserTests.wiki

```diff
@@ -1,8 +1,8 @@
----
+﻿---
```

Comment:
This change of the first line (not quite sure what it is) breaks the page on my machine. Can you revert that?

Reply:
Ok. Don't know what the difference is but will try to revert.